### PR TITLE
tests: add new testing file for alert presentable (SDKCF-6375)

### PR DESCRIPTION
**Pull request by ts-robih-robihamanto**
# Description
- Add a new testing file for AlertPresentable file covering 100% code coverage for this file.
- This PR also covers displaying an alert issue that alerts not showing because of the view being dismissed. I captured the strong reference inside the closure to keep the view to show the alert.

## Links
SDKCF-6375

# Checklist
- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [x] I have added to the [changelog](../blob/master/CHANGELOG.md#Unreleased)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data **before every commit**, including API endpoints and keys, and internal links
- [ ] I ran `fastlane ci` without errors
- [x] All project file changes are replicated in `SampleSPM/SampleSPM.xcodeproj` project

### CODE REVIEW

File: Tests/UITests/AlertPresentableSpec.swift [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            app.launch()
+        }
+
+        beforeEach {
+            mockServer = MockServerHelper.setupNewServer(route: MockServerHelper.standardRouting)
+        }
+
+        afterEach {
+            mockServer.stop()
+            app = nil
```
Comment (imaginaris) [inline Tests/UITests/AlertPresentableSpec.swift]: In other UITest specs the condition in line 17 looks like this:
`guard app == nil || !app.launchArguments.joined(separator: " ").contains(context) else {`
Thanks to that, there's no need to restart the app with `app = nil` (which takes time).
Did you try to use this approach here?

File: Tests/UITests/AlertPresentableSpec.swift [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                        app.buttons["purchase_successful"].tap()
+                        expect(iamView.exists).toEventually(beTrue(), timeout: .seconds(2))
+                    }
+                }
+
+                it("should show error alert after tapping button 1 and close campaign") {
+                    content.tap()
+                    expect(app.alerts.element.staticTexts["Page not found"].exists).to(beTrue())
+                    app.alerts.element.buttons["Close"].tap()
+                    iamView.buttons["exitButton"].tap()
```
Comment (imaginaris) [inline Tests/UITests/AlertPresentableSpec.swift]: tapping the content doesn't close the campaign? (Are those two lines necessary?)

File: Tests/UITests/ResponseStubs/slide-up-trigger-invalid-url.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{
+    "currentPingMillis": 0,
+    "nextPingMillis": 3600000,
+    "data": [
+        {
+            "campaignData": {
+                "campaignId": "slide-up-trigger",
```
Comment (imaginaris) [inline Tests/UITests/ResponseStubs/slide-up-trigger-invalid-url.json]: please use "slide-up-trigger-invalid-url" as campaignId (just in case)

Comment (ts-robih-robihamanto) [reply Tests/UITests/ResponseStubs/slide-up-trigger-invalid-url.json]: Updated here [e9fcb20](https://github.com/rakutentech/ios-inappmessaging/pull/269/commits/e9fcb2087c423e15b3dcd70d4445dd2b56505760).

File: Tests/UITests/AlertPresentableSpec.swift (hunk 1 above)
Comment (ts-robih-robihamanto) [reply Tests/UITests/AlertPresentableSpec.swift]: Yes, I should use this approach. Updated here [f2b229d](https://github.com/rakutentech/ios-inappmessaging/pull/269/commits/f2b229de48740aada9365a28a8f1438f8ef8ca8d).

File: Tests/UITests/AlertPresentableSpec.swift (hunk 2 above)
Comment (ts-robih-robihamanto) [reply Tests/UITests/AlertPresentableSpec.swift]: It is not necessary, removed here [00a6b6a](https://github.com/rakutentech/ios-inappmessaging/pull/269/commits/00a6b6aaefe88e6b43e71bb8da6dc3997aa1f3f8).

**Review by imaginaris — APPROVED** [review]

**Pull request merged**